### DISCUSSION TITLE
msfconsole run problem

### DISCUSSION THREAD

**Discussion by GwonTaeWoo** [rapid7/metasploit-framework#15394] [Q&A]
I want install Automation Tool in Kali. 
But I have Problem. This Massages

1. Initial Step:  ./docker/bin/msfconsole
I can See this Messages
**ERROR: This script does not work on Python 2.7 The minimum supported Python version is 3.6. Please use https://bootstrap.pypa.io/pip/2.7/get-pip.py instead.
ERROR: Service 'ms' failed to build : The command '/bin/sh -c python get-pip.py' returned a non-zero code: 1**

So I reinstall Python3.6 but How can I do Next.

**Comment by riz4d** [comment]
```bash
sudo apt update && sudo apt install python3.6 && sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1 && python3 --version
```

The above command will update your system, and install Python 3.6, set it as the default Python version. Then check your Python version.
